Sources/App/Views/Sections/Editor/EditorSection.swift

```diff
@@ -14,15 +14,18 @@ struct EditorSection: View {
   /// The app environment object.
   @EnvironmentObject var appEnvironment: AppEnvironment
 
+  /// The editor environment object.
+  @EnvironmentObject var editorEnvironment: EditorEnvironment
+
   // MARK: - Body
 
   var body: some View {
     NavigationView {
       Sidebar(selectedSection: $appEnvironment.selectedSection)
 
-      SourceTree(viewModel: try! SourceTreeViewModel())
+      SourceTree(viewModel: SourceTreeViewModel(editorEnvironment: editorEnvironment))
```

Comment:
If we will change the environment to `EditorSectionEnvironment` we should change this parameter too

```suggestion
      SourceTree(viewModel: SourceTreeViewModel(editorSectionEnvironment: editorSectionEnvironment))
```